web/src/App.tsx

```diff
@@ -40,10 +40,16 @@ function NewTabLink({ children, href }: { children: ReactNode; href: string }) {
 	);
 }
 
+const initialHistory = JSON.parse(localStorage.getItem('fend_history') || '[]');
+
 export default function App({ widget = false }: { widget?: boolean }) {
 	const [currentInput, setCurrentInput] = useState('');
 	const [output, setOutput] = useState<ReactNode>(widget ? <></> : exampleContent);
-	const [history, setHistory] = useState<string[]>([]);
+	const [history, setHistory] = useState<string[]>(initialHistory);
+	useEffect(() => {
+		const history100 = history.slice(0, 100);
```

Comment:
I haven't tested this locally so I could be wrong, but doesn't this only store the first 100 entries rather than the most recent?

Reply:
Yikess, yes you're totally righty. Fixed and tested